frontend/projects/valtimo/task/src/lib/components/task-list/task-list.component.scss

```
@@ -24,3 +24,16 @@
   width: 250px;
 }
+
+.valtimo-task-list__unread-dot {
+  display: inline-block;
+  width: 8px;
+  height: 8px;
+  border-radius: 50%;
+  background-color: var(--cds-link-primary);
+}
+
+::ng-deep .valtimo-task-list__unread-column {
```

Stylelint (17.3.0)

[error] Unexpected unknown pseudo-element selector "::ng-deep" (selector-pseudo-element-no-unknown)

(selector-pseudo-element-no-unknown)